osclib/pkglistgen_comments.py

```diff
@@ -145,19 +214,41 @@ def is_approved(self, comment, comments):
                     return c['who']
         return None
 
-    def parse_title(self, line):
+    def parse_title(self, line: str) -> Optional[PkglistSection]:
+        """
+        Parses the header of a section from a single line of a comment that has been passed.
+
+        :param line: The line that should be checked.
+        :returns: None or a dict with the following structure: {"cmd": "<add|move|remove", "<to|from>": str, "pkgs": []}
```

Comment:
Did I mention why I dislike docstrings on the base of them getting old very quickly and as such never be trustworthy? What an irony you're demonstrating it live ;)

Reply:
Oh damn. Yes indeed you are correct. It seems I have to do another version of this PR. I will remove it as desired from you.